docs/source/styles.rst

```diff
@@ -0,0 +1,52 @@
+Styles
+------
+
+SectionDoc comes with a number of predefined rendering styles
```

Comment:
While there is only one predefined style, you shouldn't say "a number of", you should say "one predefined rendering style"